models/image_classification_mobilenet/mobilenet.cpp

```diff
@@ -0,0 +1,140 @@
+
+#include <opencv2/imgproc.hpp>
+#include <opencv2/highgui.hpp>
+#include <opencv2/dnn.hpp>
+#include <opencv2/core/utils/logger.hpp>
+
+using namespace cv;
+using namespace dnn;
+
+std::string keys =
+"{ help  h          | | Print help message. }"
+"{ model m             | image_classification_mobilenetv1_2022apr.onnx | An optional path to file with preprocessing parameters }"
+"{ classes             | labels_imagenet_1k.txt |path to labels file  }"
+"{ input i          | | Path to input image or video file. Skip this argument to capture frames from a camera.}"
+"{ initial_width    | 0 | Preprocess input image by initial resizing to a specific width.}"
+"{ initial_height   | 0 | Preprocess input image by initial resizing to a specific height.}"
+"{ std              | 1.0 1.0 1.0 | Preprocess input image by dividing on a standard deviation.}"
+"{ rgb              | true | swap R and B plane.}"
+"{ crop             | false | Preprocess input image by center cropping.}"
+"{ backend          | 0 | Choose one of computation backends: "
+"0: automatically (by default), "
+"1: Halide language (http://halide-lang.org/), "
+"2: Intel's Deep Learning Inference Engine (https://software.intel.com/openvino-toolkit), "
+"3: OpenCV implementation, "
+"4: VKCOM, "
+"5: CUDA, "
+"6: WebNN }"
+"{ target           | 0 | Choose one of target computation devices: "
+"0: CPU target (by default), "
+"1: OpenCL, "
+"2: OpenCL fp16 (half-float precision), "
+"3: VPU, "
+"4: Vulkan, "
+"6: CUDA, "
+"7: CUDA fp16 (half-float preprocess) }";
+
+using namespace std;
+using namespace cv;
+using namespace dnn;
+
+std::vector<std::string> classes;
+
+int main(int argc, char** argv)
+{
+    std::vector<std::string> classes;
+    CommandLineParser parser(argc, argv, keys);
+
+    parser.about("Use this script to run classification deep learning networks in opencv Zoo  using OpenCV.");
+    if (argc == 1 || parser.has("help"))
+    {
+        parser.printMessage();
+        return 0;
+    }
+
+    Scalar mean(123.675, 116.28, 103.53);
+    Scalar std(255*0.229, 255*0.224, 255*0.225);
+    double scale = 1.0;
+    Size inpSize(224, 224);
+    int rszWidth = parser.get<int>("initial_width");
+    int rszHeight = parser.get<int>("initial_height");
+    bool swapRB = parser.get<bool>("rgb");
+    bool crop = parser.get<bool>("crop");
+    String model = samples::findFile(parser.get<String>("model"));
+    int backendId = parser.get<int>("backend");
+    int targetId = parser.get<int>("target");
+
+    if (model.empty())
+    {
+        CV_Error(Error::StsError, "Model file " + model + " not found");
+    }
+    if (parser.has("classes"))
+    {
+        std::string file = parser.get<String>("classes");
+        std::ifstream ifs(file.c_str());
+        if (!ifs.is_open())
+            CV_Error(Error::StsError, "File " + file + " not found");
+        std::string line;
+        while (std::getline(ifs, line))
+        {
+            classes.push_back(line);
+        }
+    }
```

Comment:
Please move the labels to the end of this file and replace this with a call to get labels.